src/node.cc

```diff
@@ -3955,6 +3955,12 @@ static void ParseArgs(int* argc,
     exit(9);
   }
 
+  if (debug_options.debug_brk_used() && !debug_options.inspector_enabled()) {
+    fprintf(stderr,
+            "%s: Using --debug-brk without --inspect is no longer supported\n", argv[0]);
```

Comment:
It should also be a proper deprecation warning... e.g.

```js
process.emitWarning(message, 'DeprecationWarning', 'DEP00NN');
```

Where `DEP00NN` is an appropriate deprecation code (see `docs/api/deprecations.md`)